src/components/badges/_badges.scss

```diff
@@ -4,7 +4,7 @@ $includeHtml: false !default;
 
   .sg-badge {
     display: inline-block;
-    border-radius: gutter(1 / 6);
+    border-radius: $defaultBorderRadius / 2;
```

Comment:
Ok, I will create 4 different borders in config then and refactor. Any other suggestions?